dlk/python/dlk/templates/src/matrix/multiplication.cpp

```diff
@@ -80,7 +80,7 @@ void matrix_multiplication_impl(
   for (std::size_t j = 0; j < B.cols(); j += regblock_m) {
     if (B.cols() - j >= regblock_m) {
```

Comment:
This might be out of scope of this PR, is there a reason to write the condition in this way?

For me, `j + regblock_m < B.cols()` looks natural and consistent to line 80.

Reply:
Fixed.